src/util/SqlTagUtils.ts

```diff
@@ -0,0 +1,49 @@
+import { Driver } from "../driver/Driver"
+
+export function buildSqlTag(params: {
+    driver: Driver
+    strings: TemplateStringsArray
+    expressions: unknown[]
+}): { query: string; variables: unknown[] } {
```

Comment:
Readability could be improved by destructuring `params`, i.e.

```ts
export function buildSqlTag({ driver, strings, expressions }: {...}
```

Reply:
Created an interface and destructed `params`